src/main/webapp/dataverse.xhtml

```diff
@@ -1,530 +1,230 @@
-                                                                         oncomplete="javascript:bind_bsui_components();">
-                                                            <f:param name="SKIP_VALIDATION" value="true"/>
-                                                            <h:outputText styleClass="glyphicon glyphicon-minus no-text"/>
-                                                        </p:commandLink>
-                                                    </div>
-                                                </div>
-                                            </ui:repeat>
-                                        </div>
-                                    </p:fragment>
-                                </div>
-                            </div>
-                            <div class="col-xs-6 form-group form-col-container">
-                                <label class="col-sm-3 control-label" for="description">
-                                    #{bundle.description}
-                                    <span class="glyphicon glyphicon-question-sign tooltip-icon"
-                                          data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['dataverse.description.title']}"></span>
-                                </label>
-                                <div class="col-sm-9 form-col-container">
-                                    <p class="help-block">
-                                        <h:outputFormat value="#{bundle.htmlAllowedMsg}" escape="false">
-                                            <f:param value="#{bundle.htmlAllowedTags}"/>
-                                        </h:outputFormat>
-                                    </p>
-                                    <p:inputTextarea id="description" tabindex="6" styleClass="form-control" value="#{DataversePage.dataverse.description}"
-                                                    rows="6" cols="50"
-                                                    autoResize="false">
-                                    </p:inputTextarea>
-                                    <p:message for="description" display="text"/>
-                                </div>
+      xmlns:jsf="http://xmlns.jcp.org/jsf"
+      xmlns:f="http://xmlns.jcp.org/jsf/core">
+<h:head>
+</h:head>
+
+<h:body>
+    <ui:composition template="/dataverse_template.xhtml">
+        <ui:param name="pageTitle" value="#{DataversePage.dataverse.name}"/>
+        <ui:param name="dataverse" value="#{DataversePage.dataverse}"/>
+        <ui:define name="body">
+            <f:metadata>
+                <f:viewParam name="id" value="#{DataversePage.dataverse.id}"/>
+                <f:viewParam name="alias" value="#{DataversePage.dataverse.alias}"/>
+                <f:viewParam name="ownerId" value="#{DataversePage.ownerId}"/>
```

Comment:
Now i realize that passing ownerId as query param is obsolete now and can be removed

Reply:
With removed DataversePage.ownerId field this line will probably cause a crash

Reply:
That's why in the new commit it is removed :P https://github.com/CeON/dataverse/pull/136/commits/a8e8c26748ccdc004b37131b215757cd8ce9c696